paper/paper.md

```diff
@@ -0,0 +1,137 @@
+  - name: Erik Faulhaber
+    orcid: 0000-0001-9788-5949
+    equal-contrib: true # (This is how you can denote equal contributions between multiple authors)
+    affiliation: 2
+  - name: Sven Berger
+    orcid: 0000-0001-6083-7038
+    equal-contrib: true # (This is how you can denote equal contributions between multiple authors)
+    affiliation: 3
+  - name: Michael Schlottke-Lakemper
+    orcid: 0000-0002-3195-2536
+    affiliation: 1 # TODO: @Michael: Stuttgart or Augsburg
+  - name: Gregor J. Gassner
+    orcid: 0000-0002-1752-1158
+    affiliation: 2
+affiliations:
+ - name: High-Performance Computing Center Stuttgart, University of Stuttgart, Germany
+   index: 1
+ - name: Department of Mathematics and Computer Science, University of Cologne, Germany
+   index: 2
+ - name: Institute of Surface Science, Helmholtz-Zentrum hereon, Germany
+   index: 3
+date: 27 March 2024 # TODO
+bibliography: paper.bib
+
+# Optional fields if submitting to a AAS journal too, see this blog post:
+# https://blog.joss.theoj.org/2018/12/a-new-collaboration-with-aas-publishing
+aas-doi: 10.3847/xxxxx <- update this with the DOI from AAS once you know it.
+aas-journal: Astrophysical Journal <- The name of the AAS journal.
+---
+
+# Summary
+
+Numerical simulations such as computational fluid dynamics, structural mechanics, thermodynamics
+or magnetohydrodynamics are often simulated using mesh-based methods. A wide range of
+mature and powerful software is available for these simulations. However, in certain applications,
+these methods reach their limits or are not designed to solve these problems efficiently.
+This is where particle-based methods offer an alternative.
+
+The main applications of particle-based methods such as Smoothed Particle Hydrodynamics (SPH),
+include hydrodynamics for the simulation of fluids with free surfaces, multiphase flows with
+complex interfaces, solid mechanics and also the simulation of heat conduction problems.
+
+The mesh-free formalism of the methods allows for easy preprocessing, making them particularly
+suitable for simulating complex geometries. This also facilitates the coupling of different single
+physics into a multiphysics system.
+
+# Statement of need
+
+TrixiParticles.jl is an open-source numerical simulation framework for
+particle-based multiphysics simulations, implemented in Julia as part of the modern,
+Julia-based Trixi framework [@trixi].
+Our primary goal is to provide a user-friendly application, accessible even to
+those unfamiliar with particle-based methods. To this end, TrixiParticles.jl is designed
+for easy extensibility, enabling researchers to experiment with new models or methods quickly,
+without the need to study and modify large parts of the code extensively.
+We strive to achieve the highest possible performance without compromising on readability
+and ease of use. This approach allows users not only to prototype new ideas rapidly,
+but also to scale up simulations efficiently.
+
```

Comment:
Misses "A description of how this software compares to other commonly-used packages in this research area."
Best list all large **active** (release or commit within the last 12 month) open source projects. Differentiation points could be for example: HPC (pysph, splishsplash), material science (splishsplash), methods besides SPH (most of them).

Reply:
All of them using TLSPH and coupling methods for FSI.
I could add a table?
| Open-source software          |Programming language  |  GPU support |
|-------------------------------|----------------------|--------------|
| TrxiParticles.jl              | Julia                | yes          |
| PySPH [@PySPH]                | Python               | no           |
| DualSPHysics [@DualSPHysics]  | C++                  | yes          |
| SplisHSPlasH [@SplisHSPlasH]  | C++                  | yes          |
| SPHinXsys [@SPHinXsys]        | C++                  | yes          |

Reply:
Yes a table is probably good since it also doesn't count against the word count.
To add some variety:
https://yade-dem.org/doc/
The only other large opensource DEM code is LIGGGHTS which seems to be dead.

Reply:
There are a few DEM-codes available, but they are all either just DEM solvers or CFDEM, i.e. coupling with mesh-based methods. I haven't found any software that includes both SPH and DEM. Does it make sense to include DEM code in the table?

Note that we only have a DEM prototype and no coupling with SPH yet.

Reply:
Hmm. Than that might be a good point that there is no sph-dem code available as something that can be mentioned as an outlook.

Reply:
I forgot this one:
https://github.com/KratosMultiphysics/Kratos

Most of the other ones seem to be not what we want to do or are dead.

Reply:
Yes but its better for us to have different methods than doing the same.

Reply:
There should also be one feature in the table that is a clear differentiation for us